functions/api/reviewsAgg.ts

```diff
@@ -0,0 +1,108 @@
+const main = async (argumentJson) => {
```

Comment:
are you able to place this under `functions/http`. That is the new convention for naming this directory